osf/management/commands/test_publish.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand
+from osf.models import OSFUser
+from osf.external.messages.celery_publishers import (
+    publish_deactivated_user,
+    publish_reactivate_user,
+    publish_merged_user,
+)
+
+
+class Command(BaseCommand):
+    help = 'Sends a message to manage a user state, for test purposes only.'
+
+    def add_arguments(self, parser):
```

Comment:
missing support to provide `merged_user_uri` in the case of a `merge` action

Reply:
It's somewhat deliberate choice to ensure data integrity. Here's the function it's hard coded to always be the `merged_by`
```
def publish_merged_user(user):
    assert user.merged_by, 'User received merge signal, but has no `merged_by` reference.'
    enqueue_task(
        _publish_user_status_change.s(
            body={
                'action': 'merge',
                'user_uri': user.get_semantic_iri(),
                'merged_user_uri': user.merged_by.get_semantic_iri(),
            },
        )
    )
```